pkg/cmd/extension/init/params.go

```diff
@@ -42,10 +42,10 @@ func (p *param) IsValid() bool {
 
 // params represents all parameters to be filled in by a user.
 type params struct {
-	Category  param
-	Language  param
-	OutputDir param
-	Name      param
+	Category     param
+	Language     param
+	ExtensionDir param
+	Name         param
```

Comment:
no need to export these fields 😄

Reply:
same goes for `param` struct above 😄

Reply:
normally I would, but some of these are used in templates and don't want to clean that up right now if there's a reference miss. I'll add a TODO